packages/backend-lalr/happy-backend-lalr.cabal

```diff
@@ -47,11 +47,14 @@ library
                        Happy.Backend.LALR.ProduceCode
   build-depends:       base < 5,
                        array,
+                       mtl > 2,
+                       containers,
                        happy-codegen-common == 1.21.0,
                        happy-grammar == 1.21.0,
-                       happy-tabular == 1.21.0
+                       happy-tabular == 1.21.0,
+                       happy-code-combinators == 1.21.0
 
   default-language:    Haskell98
-  default-extensions:  CPP, MagicHash, FlexibleContexts
+  default-extensions:  CPP, MagicHash, FlexibleContexts, TypeApplications, OverloadedStrings
```

Comment:
`TypeApplications` can be a problem, since it only exists since 8.0.1, while this package claims compatibility with 7.0.4. I think we agreed to figure out what to do about this later, but to make things easier in the future, I think it would be better not to enable it here package-wide and instead explicitly enable in the files that require it.